fsthttp/backend.go

```diff
@@ -311,6 +311,49 @@ func (b *BackendOptions) PoolConnections(poolingOn bool) *BackendOptions {
 	return b
 }
 
+// HttpKeepaliveTime configures how long to allow HTTP connections to remain
+// idle in a connection pool before it should be considered closed.
+func (b *BackendOptions) HttpKeepaliveTime(time time.Duration) *BackendOptions {
+	b.abiOpts.HttpKeepaliveTime(time)
+	return b
+}
+
+// TcpKeepaliveEnable sets whether or not to use TCP keepalives to try to
+// maintain the connetion to the backend.
+func (b *BackendOptions) TcpKeepaliveEnable(enable bool) *BackendOptions {
+	b.abiOpts.TcpKeepaliveEnable(enable)
+	return b
+}
+
+// TcpKeepaliveInterval sets the interval to use when sending TCP keepalive
+// probes. Setting this value implicitly enables TCP keepalives. Intervals of
+// less than 1 second will be rounded up to 1 second.
```

Comment:
I don't know if it's better to just document it like this or to make the order irrelevant, but it feels like this could end up mattering:

```suggestion
// TcpKeepaliveInterval sets the interval to use when sending TCP keepalive
// probes. Intervals of less than 1 second will be rounded up to 1 second.
//
// Setting this value implicitly enables TCP keepalives. If you are calling both
// this method and `TcpKeepAliveEnable` with dynamically loaded or generated
// values, make sure to call `TcpKeepAliveEnable` last.
```